RadixWallet.xcodeproj/project.pbxproj

```diff
@@ -8787,10 +8764,13 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_STYLE = Automatic;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
```

Comment:
We should probably continue using Config files, instead of modifying the pbxproj directly

Reply:
yeah I have a branch for that, but it is a bigger task, so lets do it later.